lib/cloud-config-manager.ts

```diff
@@ -1,10 +1,10 @@
 import * as path from "path";
 
-export class ServerConfigManager implements IServerConfigManager {
+export class ServerConfigManager implements ICloudConfigManager {
```

Comment:
Shouldn't you rename this to "CloudConfigManager"?

Reply:
🎣